imap_processing/tests/glows/test_glows_l1a_data.py

```diff
@@ -361,3 +391,120 @@ def test_generate_status_data():
 
     output = StatusData(test2)
     assert dataclasses.asdict(output) == expected
+
+
+def test_expected_results(l1a_test_data):
+    _, de_data = l1a_test_data
+
+    # Validation data is generated from the code sent over by GLOWS team. Contains the
+    # first 20 packets
+    validation_data = pd.read_csv(
+        Path(__file__).parent / "direct_events_validation_data_l1a.csv"
+    )
+
+    for index in validation_data.index:
```

Comment:
It might be good to add an assertion that `len(validation_data) == 20` before this loop to make sure we are actually iterating through something. Otherwise we could just pass over this loop without any assertions and still pass the test if validation_data was empty.